subprojects/docs/src/samples/java/jvm-multi-project-with-test-aggregation-standalone/groovy/test-results/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'base'
+    id 'test-report-aggregation'
+}
+
+//repositories {
+//    mavenCentral()
+//}
+
+dependencies {
+    testReportAggregation project(':application') // <1>
+}
+
+reporting {
+    reports {
+        testAggregateTestReport(AggregateTestReport) { // <2>
+            testType = TestType.UNIT_TESTS
+        }
+    }
+}
+
+// Optional: make aggregate test report generation part of the 'check' lifecycle phase
+tasks.named("check") {
+    dependsOn tasks.named('testAggregateTestReport', TestReport)
```

Comment:
Why the 2nd arg here?

Reply:
Habit. I know it provides no value when creating a simple dependency from a lifecycle task, but it also has illustrative purpose showing that the `testAggregateTestReport` has type `TestReport`.